pandas/tests/reshape/test_wide_to_long.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+import pandas as pd
+from pandas.util import testing as tm
+import pytest
+import numpy as np
+
+
+@pytest.fixture
+def buggy_wide_df():
+    #Here, we have the rare case that a column name (A) is a substring
```

Comment:
lint issues here

Reply:
Thank you! I followed all the links to the pep8speaks output and made changes accordingly.